components/Headers/SubHeader.tsx

```diff
@@ -0,0 +1,21 @@
+import styles from "./SubHeader.module.css";
+
+export type SubHeaderProps = {
+  size: "big" | "small";
+  label: string;
+};
+
+function SubHeader({ label, size }: SubHeaderProps) {
+  const theSize =
+    size === "big"
+      ? `${styles.subHeader} ${styles.big}`
+      : `${styles.subHeader}`;
+
+  return (
+    <>
+      <div className={theSize}>{label}</div>
+    </>
```

Comment:
```suggestion
      <div className={theSize}>{label}</div>
```
You don't need fragments here as it is a single element.

Reply:
tx :)